Makefile

```diff
@@ -9,12 +9,19 @@ IMAGENAME ?= resource-topology-exporter
 IMAGETAG ?= latest
 RTE_CONTAINER_IMAGE ?= quay.io/$(REPOOWNER)/$(IMAGENAME):$(IMAGETAG)
 
+VERSION := $(shell git tag --sort=committerdate | head -n 1)
+ifeq ($(VERSION),)
+	VERSION = v0.1
```

Comment:
in case we can't detect the version from the git history, better to use something clearly identifiable like `v0.0.0` or `-devel`. But we can improve later.